modules/postgres/postgres_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/docker/go-connections/nat"
 	"github.com/jackc/pgx/v5"
 	_ "github.com/lib/pq"
+
```

Comment:
The lint is complaining here: could you please run `make lint` in the postgres module dir? 🙏